es/content_units_index.go

```diff
@@ -326,6 +325,9 @@ func (index *ContentUnitsIndex) prepareIndexUnit(cu *mdbmodels.ContentUnit, inde
 			if val, ok := indexData.Sources[cu.UID]; ok {
 				unit.FilterValues = append(unit.FilterValues, KeyValues(consts.ES_UID_TYPE_SOURCE, val)...)
 				unit.TypedUids = append(unit.TypedUids, KeyValues(consts.ES_UID_TYPE_SOURCE, val)...)
+				//  We dont add TitleSuggest to CU with source
+			} else {
+				unit.TitleSuggest = Suffixes(i18n.Name.String)
```

Comment:
Are we skipping title suggest for a unit if it is not a source?

Reply:
yes, if the CU is related to a source then we skip the suggest. The suggest for the source itself is enough to find the CU also.